src/compile/layout/header.ts

```diff
@@ -84,6 +84,28 @@ export function getTitleGroup(model: Model, channel: HeaderChannel) {
   };
 }
 
+export function getLabelAlign(angle: number) {
+  angle %= 360;
```

Comment:
% in javascript can still produce negative values.  -- you may need to modify this line a bit (see what we do with `axis` code)

Reply:
Interesting..